test/mission-network-isolation.test.js

```diff
@@ -64,38 +85,79 @@ describe('Sensors test', () => {
   // After doing this, the variable notificationPayload will hold the body. On the Assert phase, ensure that it contains the right schema or data
+  test('When emitting a new event and the notification service, then the added event was still saved successfully', async () => {
+    // Arrange
+    const eventToAdd = getSensorEvent({
+      temperature: 80, //💡 TIP: We need high temperature to trigger notification
+      notificationCategory: getShortUnique(), //💡 TIP: Unique category will lead to unique notification URL. This helps in overriding the nock
+    });
+    // 💡 TIP: Set here a nock that replies with 500: nock('http://localhost').post(`/notification/${eventToAdd.notificationCategory}`)
+    let notificationPayload;
+    const scope = nock('http://localhost').post(`/notification/${eventToAdd.notificationCategory}`,
+        (payload) => (notificationPayload = payload),
+      ).reply(200, {success: true,});
+
+
+    // Act
+    const receivedResponse = await request(expressApp)
+      .post('/sensor-events')
+      .send(eventToAdd);
+
+    // Assert
+    // 💡 TIP: It's not about the response rather about checking that it was indeed saved and retrievable
+    // 💡 TIP: Whenever possible always use a public API/REST and not a direct call the DB layer
+    expect(notificationPayload).toMatchObject({
+      title: expect.any(String),
+      id: expect.any(Number),
+    });
+  });
 
   // ✅ TASK: Write the following test below
   test('When emitting a new event and the notification service replies with 500 error, then the added event was still saved successfully', async () => {
     // Arrange
+    const notificationCategory = getShortUnique();
     const eventToAdd = getSensorEvent({
       temperature: 80, //💡 TIP: We need high temperature to trigger notification
-      notificationCategory: getShortUnique(), //💡 TIP: Unique category will lead to unique notification URL. This helps in overriding the nock
+      notificationCategory: notificationCategory, //💡 TIP: Unique category will lead to unique notification URL. This helps in overriding the nock
     });
     // 💡 TIP: Set here a nock that replies with 500: nock('http://localhost').post(`/notification/${eventToAdd.notificationCategory}`)
+    nock('http://localhost').post(`/notification/${eventToAdd.notificationCategory}`,
+        (payload) => (notificationPayload = payload),
+      ).reply(500);
+    const dataAccessMock = sinon.mock(SensorsDal.prototype);
+    eventToAdd.notificationSent = false;
+    dataAccessMock.expects("addSensorsEvent").exactly(1).withArgs(eventToAdd).resolves(true);
+
 
     // Act
+    const receivedResponse = await request(expressApp)
+      .post('/sensor-events')
+      .send(eventToAdd);
 
     // Assert
     // 💡 TIP: It's not about the response rather about checking that it was indeed saved and retrievable
     // 💡 TIP: Whenever possible always use a public API/REST and not a direct call the DB layer
+    dataAccessMock.verify();
```

Comment:
is it possible to use the real DB and instead use another API call to get the values and check if the event was actually added?